src/helpers/spaces.ts

```diff
@@ -145,6 +145,51 @@ async function getVotes() {
   return await db.queryAsync(query);
 }
 
+export async function getCombinedMembersAndVoters(spaceId: string, cursor: string | null, pageSize: number, knownAdmins: string[] = [], knownModerators: string[] = [], knownMembers: string[] = []) {
+  const params: (string | number)[] = [];
+  let subqueries: string[] = [];
+
+  // Excluding known addresses fetched during the Space verification
+  const exclusionList = [...knownAdmins, ...knownModerators, ...knownMembers];
+  let placeholders = exclusionList.map(() => '?').join(', ');
+
+  // Other roles are already known and fetched at the app level while Space Verification
+  subqueries.push(`
+    SELECT DISTINCT voter AS address
+    FROM votes
+    WHERE space_id = ? AND voter NOT IN (${placeholders})
```

Comment:
There is no such field as `space_id`, it should be `space` 
```suggestion
    WHERE space = ? AND voter NOT IN (${placeholders})
```